sensorlib/config/simulated_sensor_config.yaml

```diff
@@ -19,7 +19,7 @@ simulated_sensor:
     min_sample_count: 30
     downsample_ratio: 2
     geometry_association_max_dist_in_meters: 3.0
-  use_sensor_centric_frame: False
+  use_sensor_centric_frame: True
 lidar_sensor:
   lower_fov: -80.0
   upper_fov: 30.0
```

Comment:
Can we actually modify the default noise values of `position_in_meters` and `linear_velocity_in_ms` of the sensor from 0;.8 to 0.4? Here:
https://github.com/usdot-fhwa-stol/carma-utils/blob/develop/sensorlib/config/noise_model_config.yaml#L10-L13

It is unrealistic for our scenario for the lidars to have a noise of 80cm big in close distances of 2-4 meters. Let's change it to 0.2 on all instances of 0.8